homeassistant/components/incomfort/config_flow.py

```diff
@@ -63,9 +78,60 @@ async def async_try_connect_gateway(
     return None
 
 
+class InComfortOptionsFlowHandler(OptionsFlow):
+    """Handle InComfort Lan2RF gateway options."""
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Initiate options flow."""
+        return await self.async_step_gateway()
+
+    async def async_step_gateway(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Manage the options."""
+        errors: dict[str, str] | None = None
+        if (
+            user_input is not None
+            and (
+                errors := await async_try_connect_gateway(
+                    self.hass, dict(self.config_entry.data)
+                )
+            )
+            is None
```

Comment:
Why do we actually try to connect to the gateway? We don't change configuration right?